structuralcodes/geometry/profiles/_upe.py

```diff
@@ -0,0 +1,133 @@
+"""UPN profiles."""
```

Comment:
Please update the docstring.
```suggestion
"""UPE profiles."""
```